packages/gatsby-source-shopify/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "gatsby-source-shopify",
+  "version": "2.0.1",
+  "description": "Gatsby source plugin for building websites using Shopfiy as a data source.",
+  "scripts": {
+    "build": "babel src --out-dir .",
+    "prepare": "cross-env NODE_ENV=production npm run build",
+    "watch": "npm run build -- --watch"
+  },
+  "homepage": "https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-source-shopfiy#readme",
+  "repository": "https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-source-shopify",
+  "bugs": {
+    "url": "https://github.com/gatsbyjs/gatsby/issues"
+  },
+  "keywords": [
+    "gatsby",
+    "gatsby-plugin",
+    "gatsby-source-plugin",
+    "shopify"
+  ],
+  "author": "Angelo Ashmore",
+  "license": "MIT",
+  "peerDependencies": {
+    "gatsby": ">2.0.0-alpha"
+  },
+  "devDependencies": {
```

Comment:
If you follow the above approach (adding the babel preset) need to run `yarn add babel-preset-gatsby-package --dev`